docs/user-guide/synthetic.rst

```diff
@@ -0,0 +1,113 @@
+.. _synthetic:
+
+*****************
+Synthetic Packets
+*****************
+
+Overview
+========
+It is sometimes necessary or useful to generate synthetic CCSDS packets with known data for testing and validation.
+Both `~ccsdspy.FixedLength` and `~ccsdspy.VariableLength` packet types provide this functionality through their respective
+:py:meth:`ccsdspy.FixedLength.to_file` and :py:func:`ccsdspy.VariableLength.to_file` methods.
+You'll first have to define the packet as usual as if you are going to parse a file with packets in it.
+
```

Comment:
Can you add a table of contents here, like we have on the other pages? 

```
.. contents::
   :depth: 2
```

Reply:
Not sure what you mean. At the head of this page?

Reply:
Oh i see. I hadn't noticed that you'd make that changed on the other pages (also it's missing from the CCSDS page). It's a little weird since you are putting it after the overview but then it lists the overview as a section which is above the table of contents. Anyway, you're the boss! I made the change.

Reply:
Sorry for adding that in without opening a merge request, I figured you were busy and I didn't want to bother you with small things. Would you prefer to have a voice in reviewing small changes to the docs like this in the future? 

I think you're right having the overview in the TOC is a bit awkward. How about we either move the TOCs to the top on all pages,  or remove the overview as a section header ([eg here](https://docs.ccsdspy.org/en/latest/user-guide/converters.html) there is no overview section). What do you think?